src/plugins/discord/preload.ts

```
@@ -0,0 +1,130 @@
+      let avatar: string | null = null;
+
+      const accountButton =
+        document.querySelector<HTMLImageElement>(
+          'ytmusic-settings-button img#img',
+        ) ||
+        document.querySelector<HTMLImageElement>(
+          'ytmusic-settings-button yt-img-shadow img',
+        ) ||
+        document.querySelector<HTMLImageElement>(
+          'ytmusic-settings-button img',
+        );
```

ESLint

[error] Replace `⏎··········'ytmusic-settings-button·img',⏎········` with `'ytmusic-settings-button·img'`

(prettier/prettier)